### PR TITLE
Add remote debug port

### PR DESCRIPTION
**Pull request by harshach**
### Describe your changes :

I worked on the ..... because ...

#
### Type of change :

- [x] Bug fix
- [x] Improvement
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: bin/openmetadata-server-start.sh [hunk 1]
```diff
@@ -111,7 +111,7 @@ fi
 
 #Application classname
 APP_CLASS="org.openmetadata.service.OpenMetadataApplication"
-
+echo $OPENMETADATA_OPTS
```
Comment (akash-jain-10) [inline bin/openmetadata-server-start.sh:114]: Is this echo intended here? If yes, do we want to add any additional information to this echo line here ?

File: docker/local-metadata/docker-compose.yml [hunk 2]
```diff
@@ -59,6 +59,7 @@ services:
     container_name: openmetadata_server
     environment:
       # Elasticsearch configuration
+      OPENMETADATA_DEBUG: ${OPENMETADATA_DEBUG:-false}
```
Comment (akash-jain-10) [inline docker/local-metadata/docker-compose.yml:62]: Can we add this for all the docker-compose.yml files ?
development -
- docker/local-metadata/docker-compose-postgres.yml

production -
- docker/metadata/docker-compose.yml
- docker/metadata/docker-compose-postgres.yml

File: docker/local-metadata/docker-compose.yml [hunk 3]
```diff
@@ -125,9 +126,11 @@ services:
     expose:
       - 8585
       - 8586
+      - 5005
```
Comment (akash-jain-10) [inline docker/local-metadata/docker-compose.yml:129]: Can we add this for all the docker-compose.yml files ?
development -
- docker/local-metadata/docker-compose-postgres.yml

production -
- docker/metadata/docker-compose.yml
- docker/metadata/docker-compose-postgres.yml

File: docker/local-metadata/docker-compose.yml [hunk 4]
```diff
@@ -125,9 +126,11 @@ services:
...
     ports:
       - "8585:8585"
       - "8586:8586"
+      - "5005:5005"
```
Comment (akash-jain-10) [inline docker/local-metadata/docker-compose.yml:133]: Can we add this for all the docker-compose.yml files ?
development -
- docker/local-metadata/docker-compose-postgres.yml

production -
- docker/metadata/docker-compose.yml
- docker/metadata/docker-compose-postgres.yml

**Review by akash-jain-10 — CHANGES_REQUESTED** [review]

File: docker/local-metadata/docker-compose.yml (hunk 2 above)
Comment (harshach) [reply docker/local-metadata/docker-compose.yml:62]: No only for dev

File: docker/local-metadata/docker-compose.yml (hunk 3 above)
Comment (harshach) [reply docker/local-metadata/docker-compose.yml:129]: No

**Review by akash-jain-10 — APPROVED** [review]

**Pull request merged**